twingly_search/query.py

```diff
@@ -24,77 +28,146 @@ class Query(object):
                    Assumes UTC if the datetime object has no timezone set
     """
 
-    DATETIME_FORMAT = "%Y-%m-%d %H:%M:%S"
-
     def __init__(self, client):
         """
         No need to call this method manually, instead use Client#query.
 
         :param client: the client that this query should be connected to
         """
         self.client = client
-        self.pattern = ''
-        self.language = ''
-        self._start_time = None
-        self._end_time = None
+        self.search_query = ''
+        self._language = ''
+        self._start_date = None
+        self._end_date = None
+
+    @property
+    def start_date(self):
```

Comment:
One of the reasons we built the clients was to get around some of the quirks in the search language. `start-date` (in the search language) sounds like you only set the date, when you can in fact set both date and time. The reason we named it `start_time` in the client is that it better represents what it actually is for.